metrics/pipe_analysis.yaml

```diff
@@ -0,0 +1,17 @@
+# Metric definitions in pipe_analysis
+
+stellar_w_principal_color_rms_width_gri:
```

Comment:
@laurenam I would recommend not to encode information in the metric name. Metric names should be preferentially short.  Especially the `_gri` which should be part of the metric specification instead. The same metric e.g. `stellar_locus_width` might have different specs if it's measured in `g-r` vs. `r-i` or if it uses other colors.